File: src/js/usuario.js
$(document).ready(function () {
    var user_img = document.querySelector('#user_img');
    var user_name = document.querySelector('#name');
    var usuario = JSON.parse(sessionStorage.getItem('usuario'));
    user_img.setAttribute("src", usuario.FotoPerfil);
    user_img.setAttribute("border", "none");
    user_name.innerHTML = usuario.Nombre + '\n' + usuario.PrimerApellido + '\n' + usuario.SegundoApellido;
})
    

File: Entities/Usuario.go
package Entities

type Usuario struct {
	Cedula            int    `json:"Cedula,string,omitempty"`
	Nombre            string `json:"Nombre"`
	PrimerApellido    string `json:"PrimerApellido"`
	SegundoApellido   string `json:"SegundoApellido"`
	FechaNacimiento   string `json:"FechaNacimiento,timestamp"`
	CorreoElectronico string `json:"CorreoElectronico"`
	Contrasena        string `json:"Contrasena"`
	FotoPerfil        string `json:"FotoPerfil"`
}
